pandas/tests/test_import.py

```diff
@@ -0,0 +1,14 @@
+def test_import_error():
+    # GH#26367 Verify whether Pandas import succeeds when setting filterwarnings
+    import warnings
+
+    warnings.filterwarnings("error")
+
+    try:
+        import pandas as pd  # noqa: F401
```

Comment:
Don't need the try except. Just this import is fine.

Reply:
Done